app-arch/unrar/unrar-5.6.1.recipe

```diff
@@ -32,6 +32,8 @@ ARCHITECTURES="x86_gcc2 x86 x86_64"
 PROVIDES="
 	unrar = $portVersion
 	cmd:unrar = $portVersion
+	lib:libunrar = $portVersion
+	devel:libunrar = $portVersion
```

Comment:
Why isn't `devel:libunrar` resides in it's own package?

Reply:
Yep, you are right. Keeping the static lib in the base package (instead of in a `unrar_devel` subpackage) is against the general recommendation.
I did so because I think that creating an unrar_devel package just for the static lib would have been worse than putting the static lib in the base package.
BTW, the main advantage of keeping everything in the base package is that we can decide to drop the static lib at any time, without having to insert any `REPLACES`.
Indeed, if I had created a devel package, then we would be forced to include a `REPLACES="unrar_devel"` if we later decide to retire that subpackage.

BTW, I thought about removing the static lib, but then I thought that it would be OK to keep it in the base package.

Well, I see 3 scenari:
1) We keep the static lib in the base package.
2) We drop the static lib.
3) We move the static lib to a devel subpackage.

I'm OK with 1 and 2. Scenario 3 is not my favorite. Upstream does not even make it clear which header files would be required to use the lib.
BTW, my first intention was to provide the SFX file, and then I thought that also providing the shared lib would be cool. And I ended with also providing the static lib :)

FWIW, the SFX file (which allows to create self extracting archives with the `cat` command, as we know) works like a charm :)

Reply:
I just realized that there are no downsides with creating an `unrar_devel` package for the static lib.

In case there is a consensus for scenari 2 or 3, shall I wait untl next release becomes available?